trunk/SUAVE/Methods/Geometry/Two_Dimensional/Cross_Section/Airfoil/compute_airfoil_polars.py

```diff
@@ -0,0 +1,198 @@
+
+            # Equation 6d
+            N1     = 1 + CL1max/RCL1
+
+            # Equation 6a or 6b depending on the alpha                  
+            if alpha == A0:
+                CL[i,j] = 0.0        
+            elif alpha > A0: 
+                CL[i,j] = S1*(alpha - A0)-RCL1*((alpha-A0)/(ACL1-A0))**N1        
+            else:
+                CL[i,j] = S1*(alpha - A0)+RCL1 *((A0-alpha )/(ACL1 -A0))**N1 
+
+            # Equation 7a or 7b depending on alpha
+            M    = 2.0  
+            con  = np.logical_and((2*A0-ACD1)<=alpha,alpha<=ACD1)
+            if con == True:
+                CD[i,j] = CD0  + (CD1max -CD0)*((alpha  -A0)/(ACD1 -A0))**M   
+            else:
+                CD[i,j] = 0.  
+            # ------------------------------------------------------
+            # Equations for coefficients in post-stall regime 
+            # ------------------------------------------------------               
+            # Equation 9a and b
+            F1        = 1.190*(1.0-(t_c**2))
+            F2        = 0.65 + 0.35*np.exp(-(9.0/AR)**2.3)
+
+            # Equation 10b and c
+            G1        = 2.3*np.exp(-(0.65*t_c)**0.9)
+            G2        = 0.52 + 0.48*np.exp(-(6.5/AR)**1.1)
+
+            # Equation 8a and b
+            CL2max    = F1*F2
+            CD2max    = G1*G2
+
+            # Equation 11d
+            RCL2      = 1.632-CL2max
+
+            # Equation 11e
+            N2        = 1 + CL2max/RCL2
+
+            # LIFT COEFFICIENT
+            # Equation 11a,b,c
+            if alpha > ACL1:
+                con2      = np.logical_and(ACL1<=alpha,alpha<=(92.0))
+                con3      = [alpha>=(92.0)]                   
+                if con2 == True:
+                    CL[i,j] = -0.032*(alpha-92.0) - RCL2*((92.-alpha)/(51.0))**N2
+                elif con3 == True:
+                    CL[i,j] = -0.032*(alpha-92.0) + RCL2*((alpha-92.)/(51.0))**N2
+
+            # If alpha is negative flip things for lift
+            elif alpha < 0.:  
+                alphan    = - alpha+2*A0
+                con2      = np.logical_and(ACL1<=alpha, alpha<=(92.0))
+                con3      = alpha>=(92.0)                    
+                if con2 == True:
+                    CL[i,j] = 0.032*(alphan-92.0) + RCL2*((92.-alpha)/(51.0))**N2
+                elif con3 == True:
+                    CL[i,j] = 0.032*(alphan-92.0) - RCL2*((alphan-92.)/(51.0))**N2
+
+            # DRAG COEFFICIENT
+            # Equation 12a 
+            if  alpha > ACD1:
+                CD[i,j]  = CD1max + (CD2max - CD1max) * np.sin(((alpha-ACD1)/(90.-ACD1))*90.*Units.degrees)
+
+            # If alpha is negative flip things for drag
+            elif alpha < 0.:
+                alphan    = -alpha + 2*A0
+                if alphan>=ACD1:
+                    CD[i,j]  = CD1max + (CD2max - CD1max) * np.sin(((alphan-ACD1)/(90.-ACD1))*Units.degrees)     
+
+
+    airfoil_data.lift_coefficient_sweep  = CL
```

Comment:
Maybe just "lift_coefficients" etc.? It's hard to tell what sweep means from just variable name.

Reply:
Updated with the suggestion to lift/drag coefficients

Reply:
Changed